platform_email/index.js

```diff
@@ -16,7 +16,7 @@
 
 /**
 Jazz Email service
-@author: 
+@author:
```

Comment:
Why did this file become part of this PR?

Reply:
The file was most likely used for debugging and some white space may have been added by the code editor, these changes are negligible and can be reverted